src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig

```diff
@@ -0,0 +1,105 @@
+
+<ul>
+  <li>developing collaborative initiatives to preserve media</li>
+  <li>efforts to extend capabilities and opportunities within archives</li>
+  <li>active open media standardization projects</li>
+  <li>seeking consensus in audiovisual preservation strategy</li>
+  <li>examination of open media use in film and video digitization</li>
+  <li>integration of open source tools into archival workflow</li>
+  <li>examples of cross-community collaboration and skill-sharing</li>
+  <li>developments in open media</li>
+  <li>and more</li>
+</ul>
+
+<p>Follow <a href="https://twitter.com/nttwconf">@nttwconf</a> for news and updates about No Time to Wait. Our recommended hashtag is <a href="https://twitter.com/hashtag/nttw7">#nttw7</a>.</p>
+<p>The conference will be livestreaming on the MediaArea YouTube channel <a href="http://www.youtube.com/c/MediaAreaNetOfficial">available here</a>.</p>
+
+<h2>Where</h2>
+<p><a href="https://nfa.cz/">Národní filmový archiv</a> at its theater in Prague:</a> Malešická 2706/12, 130 00 Praha 3, Czechia.</p>
+
+<p>
+{% image '@NoTimeToWaitBundle/Resources/public/img/P-768x512.jpg' %}
+  <img src="{{ asset_url }}" alt="interior image of Národní filmový archiv" class="img-responsive center-xs" title="Národní filmový archiv interior of the theater">
+{% endimage %}
+</p>
+
+<p>
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2560.1680462251616!2d14.415493751272342!3d50.083140521542994!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x470b94ee4e82dc6f%3A0xa6d97f03353c46c!2sKino%20NFA%20Ponrepo!5e0!3m2!1sen!2sus!4v1678811430423!5m2!1sen!2sus" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+</p>
+
+<h2>Travel Grants</h2>
+
+<p>We're currently working on budgeting for travel grants and will announce a travel grant application on this page and at https://twitter.com/nttwconf when they're ready.</p>
+
+<h2>Sponsorship</h2>
+
+<p>The organizers of No Time to Wait strive to sustain this conference as a free registration event through efficient and transparent budgeting and via the generosity of our hosts and sponsors. The sponsors of No Time to Wait help make the discussions, presentations, and momentum of the No Time to Wait event possible and enable the event to maintain a high level of accessibility and involvement. Sponsors shall be acknowledged via the conference's website, printed programs, and presence at the conference. Sponsors may indicate that their contribution is allocated to a specific need, such as funding travel stipends, hosting a reception, or supporting live streaming. To discuss or inquire regarding sponsorship opportunities please <a href="{{ path('ma_contact') }}">please contact us</a>.
+
+<p>Thanks to the following organizations for generously supporting No Time to Wait.</p>
+<ul>
+  <li><a href="https://reto.ch/">Reto Kromer</a></li>
```

Comment:
@retokromer fine for you?

Reply:
You can write `AV Preservation by reto.ch` also this year.